src/builtins/filters/array.rs

```diff
@@ -66,6 +66,29 @@ pub fn sort(value: Value, args: HashMap<String, Value>) -> Result<Value> {
     Ok(sorted.into())
 }
 
+/// Slice the array
+/// Use the `start` argument to define where to start (inclusive, default to `0`)
+/// and `end` argument to define where to stop (exclusive, default to the length of the array)
+/// `start` and `end` are 0-indexed
+pub fn slice(value: Value, args: HashMap<String, Value>) -> Result<Value> {
+    let mut arr = try_get_value!("slice", "value", Vec<Value>, value);
+    if arr.is_empty() {
+        return Ok(arr.into());
+    }
+
+    let start = try_get_value!("slice", "start", f64, args.get("start").unwrap_or(&0.0.into())) as usize;
```

Comment:
Forgive my curiosity, but why f64 and not an integer?

Reply:
See https://github.com/Keats/tera/issues/254#issuecomment-363062924